shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/ShivaVariantcalling.scala

```diff
@@ -48,9 +49,22 @@ class ShivaVariantcalling(val parent: Configurable) extends QScript
 
   var inputBqsrFiles: Map[String, File] = Map()
 
+  var genders: Map[String, Gender.Value] = _
+
   /** Executed before script */
   def init(): Unit = {
     if (inputBamsArg.nonEmpty) inputBams = BamUtils.sampleBamMap(inputBamsArg)
+    if (genders == null) genders = {
```

Comment:
The problem here is because it's a `var`.  This required to be able to pass the map from shiva to ShivaVariantcalling. Will check if I do it maybe an other way.